bindings/java/src/org/sleuthkit/datamodel/HostAddressManager.java

```diff
@@ -52,12 +52,19 @@
 	private final Cache<Long, Byte> recentHostNameAndIpMappingCache = CacheBuilder.newBuilder().maximumSize(200000).build();
 
 	/**
-	 * Recently added or accessed Host Address Object Ids are cached. This is
+	 * Recently added or accessed Host Address Objects are cached. This is
 	 * here to improve performance of the
 	 * {@link #hostAddressExists(org.sleuthkit.datamodel.HostAddress.HostAddressType, java.lang.String)}
-	 * check.
+	 * check as well as the {@link #getHostAddress(org.sleuthkit.datamodel.HostAddress.HostAddressType, java.lang.String) }
 	 */
-	private final Cache<String, Long> recentHostAddresstCache = CacheBuilder.newBuilder().maximumSize(200000).build();
+	private final Cache<String, HostAddress> recentHostAddresstCache = CacheBuilder.newBuilder().maximumSize(200000).build();
```

Comment:
Can you fix the existing typo here Addresst->Address

Reply:
@APriestman changes are in